corehq/apps/app_manager/templates/app_manager/partials/releases/releases_deploy_modal.html

```diff
@@ -196,51 +196,6 @@ <h4 class="panel-title">
               </div>
             </div>
           {% endif %}
-          <div class="panel panel-appmanager">
-            <div class="panel-heading">
-              <h4 class="panel-title">
-                <a data-toggle="collapse"
-                   data-bind="
-                               click: function() { track_deploy_type('Download to Java Phone'); },
-                               attr: {
-                                   href: '#' + id() + '_download_java',
-                                   'data-target': '#' + id() + '_download_java',
-                                   'data-parent': '#deploy-accordion_' + id()
-                                }
-                               "
-                   aria-expanded="true" aria-controls="panel-java">
-                  <i class="fcc fcc-applications"></i>
-                  {% trans "Download to Java Phone" %}
-                </a>
-              </h4>
-            </div>
-            <div class="panel-collapse collapse" data-bind="attr: {id: id() + '_download_java'}">
-              <div class="panel-body">
-                <ol data-bind="if: reveal_java_download()">
-                  <li>
-                    {% trans "Download <strong>both</strong> of the following files:" %}
-                    <p><a data-bind="attr: {href: $root.reverse('download_jad', id)}">{% trans "Download" %} CommCare.jad</a></p>
-                    <p><a data-bind="attr: {href: $root.reverse('download_jar', id)}">{% trans "Download" %} CommCare.jar</a></p>
-                  </li>
-                  <li>
-                    {% trans "For help on how to install, see" %}
-                    <a target="_blank" href="https://help.commcarehq.org/display/commcarepublic/CommCare+for+Feature+Phones">{% trans "Set Up Feature Phones" %}</a>.
```

Comment:
It'd be good to remove this page and related public help content. Do you think the support team could take that?

Reply:
Support has already been given the go-ahead to archive that page.